docs/docs/getting-started.md

```diff
@@ -4,82 +4,82 @@ title: Getting Started
 permalink: docs/getting-started.html
 next: tutorial.html
 redirect_from: "docs/index.html"
+redirect_from: "downloads.html"
 ---
 
-## JSFiddle
+There are several different ways to use React. The right one for you depends on how you're using it.
```

Comment:
I think people get a little frustrated when we say there are multiple ways to do the same thing. Maybe approach it from a different angle? e.g.

>React is flexible and accommodates a variety of projects. You can create new apps with it, but you can also gradually introduce it into an existing codebase without doing a rewrite.